docs/sources/helm-charts/mimir-distributed/release-notes/v5.8.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Grafana Mimir Helm chart version 5.8 release notes"
+menuTitle: "V5.8 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 5.8"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 5.8 release notes
+
+Grafana Labs is excited to announce version 5.8 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.17 and Grafana Enterprise Metrics (GEM) v2.17. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+KEDA autoscaling has changed `toPromQLLabelSelector` from an object to a list of strings, adding support for all PromQL operators.
```

Comment:
do we want to make it so that it's enough for people to read the release notes to be able to use the feature? If yes, then i'd mention the full path of the setting which is `kedaAutoscaling.toPromQLLabelSelector`